custom/enikshay/management/commands/add_back_enikshay_foreign_keys.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+from django.db import connections
+from django.core.management.base import BaseCommand
+from corehq.sql_db.util import get_db_aliases_for_partitioned_query
+
+
+def foreign_key_exists(db_alias, table_name, foreign_key_name):
+    cursor = connections[db_alias].cursor()
+    cursor.execute(
+        "SELECT 1 FROM pg_constraint WHERE conname = %s AND conrelid = %s::regclass",
+        [foreign_key_name, table_name]
+    )
+    return cursor.fetchone() is not None
+
+
+def add_locations_sqllocation_parent_fk(db_alias):
+    cursor = connections[db_alias].cursor()
+    cursor.execute(
+        "ALTER TABLE locations_sqllocation "
+        "ADD CONSTRAINT locations_sqlloc_parent_id_2ffc03fb_fk_locations_sqllocation_id "
+        "FOREIGN KEY (parent_id) REFERENCES locations_sqllocation(id) DEFERRABLE INITIALLY DEFERRED"
+    )
+
+
+def add_form_processor_xformattachmentsql_form_id_fk(db_alias):
+    cursor = connections[db_alias].cursor()
+    cursor.execute(
+        "ALTER TABLE form_processor_xformattachmentsql "
+        "ADD CONSTRAINT for_form_id_d184240c_fk_form_processor_xforminstancesql_form_id "
+        "FOREIGN KEY (form_id) REFERENCES form_processor_xforminstancesql(form_id) DEFERRABLE INITIALLY DEFERRED"
+    )
+
+
+def add_form_processor_commcarecaseindexsql_case_id_fk(db_alias):
+    cursor = connections[db_alias].cursor()
+    cursor.execute(
+        "ALTER TABLE form_processor_commcarecaseindexsql "
+        "ADD CONSTRAINT form_case_id_be4cb9e1_fk_form_processor_commcarecasesql_case_id "
+        "FOREIGN KEY (case_id) REFERENCES form_processor_commcarecasesql(case_id) DEFERRABLE INITIALLY DEFERRED"
+    )
+
+
+def add_form_processor_casetransaction_case_id_fk(db_alias):
+    cursor = connections[db_alias].cursor()
+    cursor.execute(
+        "ALTER TABLE form_processor_casetransaction "
+        "ADD CONSTRAINT form_case_id_0328b100_fk_form_processor_commcarecasesql_case_id "
```

Comment:
it's just the name that either django or postgres gives the indexes automatically